src/core/cgroup.h

```diff
@@ -9,6 +9,21 @@
 #include "list.h"
 #include "time-util.h"
 
+typedef struct TasksMax {
+        /* If scale == 0, just use value; otherwise, value / scale.
+         * See tasks_max_scale(). */
+        uint64_t value;
+        uint64_t scale;
+} TasksMax;
+
+#define TASKS_MAX_UNSET ((TasksMax) { UINT64_MAX })
```

Comment:
hmm, i really don't like pre-C99 initializers anymore i must say, would prefer if this was spelt out as .value = `UINT64_MAX` here